cli/src/main/java/org/openapitools/openapistylevalidator/cli/Main.java

```diff
@@ -41,6 +44,11 @@ public static void main(String[] args) {
     }
 
     private static void validateAndPrint(OptionManager optionManager, CommandLine commandLine) {
+        List<StyleError> result = validate(optionManager, commandLine);
+        outputUtils.printResults(result);
```

Comment:
There is no need here to use 2 lines for this and to create a variable, unless it was for clarity?